compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -263,6 +263,12 @@ int entry(int argc, char **argv)
     .default_value(false)
     .help("This will convert single input Pack to Reshape");
 
+  arser.add_argument("--substitute_pad_v2_to_pad")
```

Comment:
(optional) As `PadV2` is translated to `pad_v2` for lower case for other codes, this seems natural to convert to `substitute_pad_v2_to_pad` but when we write as `pad_v2_to_pad`, this seems a little bit strange... like `PAD - V2` sequence to `PAD` (we know there is no such thing as `V2` but someone may think as so)
so, how about `substitute_padv2_to_pad` ?

Reply:
No problem~ I will change.